code/modules/vehicles/vehicle.dm

```diff
@@ -1,7 +1,6 @@
 //Dummy object for holding items in vehicles.
 //Prevents items from being interacted with.
 /datum/vehicle_dummy_load
-	var/name = "dummy load"
```

Comment:
This isn't necessarily unused, defining a name var means datums will use it in modal inputs instead of their type.

Reply:
I think the issue is that it was also never set, so it would always show up as "dummy load".